### DISCUSSION TITLE
Ability to check passwords credentials on the fly.

### DISCUSSION THREAD

**Discussion by ceerad** [ltb-project/self-service-password#883] [Ideas]
Hello.  

I have some source code I'd like to contribute that checks the password credentials as the user types it in. We're using it on our own self-service-password instance for now, but I'm trying to write in a such a way that it contributes to the project.

Very basic example on w3c: https://www.w3schools.com/howto/howto_js_password_validation.asp

Can someone please allow me to create a branch to propose the changes. 

Thanks
Conrad

**Comment by coudot** [comment]
This feature is already coded, it will be included in next version (1.6)
